spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -934,7 +934,7 @@ The parsing of query method names follows a specific format:
 - The method name consists of the subject, the predicate, and optionally the order clause.
 - The subject begins with the action (such as `find` or `delete`) and is optionally followed by an expression (for example, `First10`), followed by any number of other characters, followed by `By`.
 - The predicate defines the query's condition or filtering criteria, where multiple conditions are delimited by `And` or `Or`. For example, `PriceLessThanAndNameLike`.
-- The order clause, which is optional, begins with `OrderBy` and consists of an ordered collection of entity attributes by which to sort results, delimited by `Asc` or `Desc` to specify the sort direction of the preceding attribute.
+- The order clause, which is optional, begins with `OrderBy` and is followed by one or more ordered pairings consisting of entity attribute name and direction of sorting (`Asc` or `Desc`). For example, `OrderByPriceDescIdAsc`. Ordering of results is performed by sorting the first entity attribute according to its respective direction, where results that sort to the same position are subsequently sorted by the second entity attribute according to its respective direction, and so forth.
```

Comment:
But, when we don't put `Asc` it is asc implicitly.

It also happens in SQL, and we share the same behavior in NoSQL:

```SQL
SELECT * FROM Products ORDER BY Price;
```

It would be great if you reproduce it here as well.

Reply:
That's a good point. The BNF does allow an optimization where the direction can be omitted on the final property and we need to define the meaning of that in the spec.  I added commit https://github.com/jakartaee/data/pull/577/commits/487098a35eabc36dfb8631dd88ffb91a17782b1a to address this.